Missing dependency on Jinja2

#### Description of problem:
Followed steps in BUILD.md  and there was a missing dependency on Jinja2.

#### SCAP Security Guide Version:
 SCAP Security Guide 0.1.39

#### Operating System Version:
FC22

#### Steps to Reproduce:

1. Followed steps in BUILD.md 
2. Error running cmake ../

#### Actual Results:
```
-- SCAP Security Guide 0.1.39
-- (see /home/omni/webCS-linux/sysapps/scap-security-guide-head/BUILD.md for build instructions)
--
CMake Error at /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:108 (message):
  Could NOT find PY_jinja2 (missing: PY_JINJA2)
Call Stack (most recent call first):
  /usr/share/cmake/Modules/FindPackageHandleStandardArgs.cmake:315 (_FPHSA_FAILURE_MESSAGE)
  cmake/FindPythonModule.cmake:23 (find_package_handle_standard_args)
  CMakeLists.txt:75 (find_python_module)


-- Configuring incomplete, errors occurred!
See also "/home/omni/webCS-linux/sysapps/scap-security-guide-head/build/CMakeFiles/CMakeOutput.log".
```

#### Expected Results:

```

-- SCAP Security Guide 0.1.39
-- (see /home/omni/webCS-linux/sysapps/scap-security-guide-head/BUILD.md for build instructions)
--
-- Found PY_jinja2: /usr/lib/python2.7/site-packages/jinja2
-- CMake:
-- generator: Unix Makefiles
-- source directory: /home/omni/webCS-linux/sysapps/scap-security-guide-head
-- build directory: /home/omni/webCS-linux/sysapps/scap-security-guide-head/build
--
-- Tools:
-- python: /usr/bin/python2 (version: 2.7.10)
-- python yaml module: /usr/lib64/python2.7/site-packages/yaml
-- python jinja2 module: /usr/lib/python2.7/site-packages/jinja2
-- oscap: /usr/bin/oscap (version: 1.2.9)
-- xsltproc: /usr/bin/xsltproc
-- xmllint: /usr/bin/xmllint
-- xmlwf: /usr/bin/xmlwf
-- sed: /usr/bin/sed
-- shellcheck (optional): /usr/bin/shellcheck
-- linkchecker (optional): LINKCHECKER_EXECUTABLE-NOTFOUND
-- grep (optional): /usr/bin/grep
--
-- Build options:
-- SSG vendor string: ssgproject
-- OVAL 5.11: ON
-- OVAL schematron validation: ON
-- shellcheck bash fixes validation: OFF
-- SVG logo in XCCDFs: ON

```
#### Addition Information/Debugging Steps:
`sudo pip install Jinja2 `
Resolved the issue

--- a/BUILD.md
+++ b/BUILD.md
@@ -4,14 +4,14 @@
 
 1. On Red Hat Enterprise Linux and Fedora make sure the packages `cmake`, `openscap-utils`, and their dependencies are installed. We require version `1.0.8` or later of `openscap-utils` (available in Red Hat Enterprise Linux) as well as `git`.
 
- `# yum -y install cmake openscap-utils git PyYAML python-ninja2`
+ `# yum -y install cmake openscap-utils git PyYAML python-jinja2`
 
  On Ubuntu, make sure the packages `expat`, `libopenscap8`, `libxml2-utils`, `xsltproc`, and their dependencies are installed as well as `git`.
 
- `$ sudo apt -y install cmake expat libopenscap8 libxml2-utils xsltproc git python-ninja2`
+ `$ sudo apt -y install cmake expat libopenscap8 libxml2-utils xsltproc git python-jinja2`
 
- On Fedora, install the python2-ninja package
- `# dnf -y install python2-ninja2`
+ On Fedora, install the python2-jinja2 package
+ `# dnf -y install python2-jinja2`
 
  Optional: Install the ShellCheck package.
 

--- a/BUILD.md
+++ b/BUILD.md
@@ -4,17 +4,20 @@
 
 1. On Red Hat Enterprise Linux and Fedora make sure the packages `cmake`, `openscap-utils`, and their dependencies are installed. We require version `1.0.8` or later of `openscap-utils` (available in Red Hat Enterprise Linux) as well as `git`.
 
- `# yum -y install cmake openscap-utils git PyYAML`
+ `# yum -y install cmake openscap-utils git PyYAML python-ninja2`
 
- On Ubuntu make sure the packages `expat`, `libopenscap8`, `libxml2-utils`, `xsltproc`, and their dependencies are installed as well as `git`.
+ On Ubuntu, make sure the packages `expat`, `libopenscap8`, `libxml2-utils`, `xsltproc`, and their dependencies are installed as well as `git`.
 
- `$ sudo apt -y install cmake expat libopenscap8 libxml2-utils xsltproc git`
+ `$ sudo apt -y install cmake expat libopenscap8 libxml2-utils xsltproc git python-ninja2`
+
+ On Fedora, install the python2-ninja package
+ `# dnf -y install python2-ninja2`
 
  Optional: Install the ShellCheck package.
 
  `# dnf -y install ShellCheck`
 
- Optional: If you want to use the Ninja build system install the ninja-build package
+ Optional: If you want to use the Ninja build system, install the ninja-build package
 
  `# dnf -y install ninja-build`
 

